markdown/ch6.markdown

```diff
@@ -333,8 +333,7 @@ layout of the worm brain neurons, they used a related matrix, the
 
 Because the order of the neurons is important in this analysis, we will use a
 preprocessed dataset, rather than clutter this chapter with data cleaning. We
-got the original data from Lav Varshney's
-[website](http://www.ifp.illinois.edu/~varshney/elegans),
+got the original data from Lav Varshney's website,
```

Comment:
Should this be a footnote, or is the URL given anywhere else?

Reply:
No, this is the only place where the URL is given. Since the original link no longer works, this might be better as an attribution footnote.

Reply:
@jni Do you know where Lav keeps these things nowadays?

Reply:
Nope, quick search helped me find his site but not the data. I'll write him an email.

In other news, what am I doing with my life? :joy: https://publish.illinois.edu/varshney/publications/

Reply:
Lav replied quickly. Good news! It's on GitHub. Should be quite a bit more stable! https://github.com/lrvarshney/elegans

Reply:
Great! I've added the link to the repo back into the text in the same format (i.e. not a footnote) as the original text - let me know what you think!